src/commands/build.rs

```diff
@@ -146,8 +151,34 @@ impl BlueBuildCommand for BuildCommand {
     fn try_run(&mut self) -> Result<()> {
         trace!("BuildCommand::try_run()");
 
-        let build_id = Uuid::new_v4();
+        // This exist to help with initial transition incase a user
+        // doesn't realize that we override the Containerfile
+        if ops::check_file_modified("Containerfile")? {
+            trace!("Containerfile is modified, checking for BlueBuild Tag");
+
+            let container_file_path =
+                std::env::current_dir().map(|p| p.join(constants::CONTAINER_FILE_PATH))?;
 
+            let file = match fs::read_to_string(container_file_path) {
+                Ok(file) => file,
+                Err(e) => {
+                    bail!("Failed to read Containerfile: {e}")
+                }
+            };
+
+            if !file.lines().any(|line| {
+                line.to_string()
+                    .trim()
+                    .starts_with(&format!("LABEL {}", constants::BUILD_ID_LABEL))
+            }) {
+                warn!("Containerfile has not been modified by BlueBuild, skipping build");
```

Comment:
These errors need to be written to be human-friendly and self-documenting. Read a bit of the CLI interface guidellnes: https://clig.dev/#errors

Here's an example:
> It looks like you have a Containerfile in this directory that has not been generated by BlueBuild. Do you want to continue building and overwrite the Containerfile with a BlueBuild-generated one (Y, n)?